README.md

```diff
@@ -1,16 +1,128 @@
-# Title
+# Git Version Control: Introduction to Version Control
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Define the purpose of a version control system
+- Recognize Benefits of Version Control Systems
+- Identify Git vocabulary terms
 
 ## Introduction
 
-## SWBAT 1
+Let's imagine we're in the job market. We're living the fabulous digital nomad
+lifestyle (working from the beaches of Koh Samui, Thailand, this time) and we
+want to land a 3-month contract for the duration of our visa.
 
-## SWBAT 2
+We're going to apply for a job and we're going to write a very simple
+text-based resume and store it in the file `my-resume-January-2017.txt`.
+
+We apply for the job, and, (congratulations!) we get it. We leave that resume
+file with other downloaded cat GIFs, receipts, and emails on our desktop for a
+few months. The job is for three months and we don't think about our resume in the
+blissful, paid weeks that go by.
+
+Three months later, we are ready to find a new contract. We have some new
+skills that we'd like to add on to our resume, but we don't want to lose our
+original resume from three months ago. We would like to create a new _version_
+of the resume.
+
+A basic way of managing a new _version_ would be to copy
+`my-resume-January-2017.txt` to `my-resume-April-2017.txt` and create a new
+_version_. We could then create new _versions_ every time we need to apply for
+a new job. And what about the case where we want to use (more-or-less) the same
+resume, but maybe want to emphasize different skills. How are we going to keep
+all those versions organized? The secret is to use a _version control system_.
+That is, software that _controls_ our collection of _versions_.
+
+## Recognize Benefits of Version Control Systems
+
+"Version Control System" describes a whole group of software. Graphics Programs
+includes Photoshop and Illustrator and MS-PAINT. Version Control System
+software include Git, Mercurial, Subversion, and others. Some other web-based
+applications like Google Docs have embraced this idea and have included
+"tracked changes" or "change history."
+
+Ultimately, while it's **awesome** to have backup of your work, one of the
+biggest benefits of version control is that you can experiment and **throw away
+bad ideas** and _instantly_ get back to your last known "good" state. Think
+about it this way, when you were learning arithmetic, you knew how to undo a
+bad calculation: turn the pencil upside down and erase. As a result you were
+**free** to try and experiment and explore knowing you could always get back to
+the original problem easily. When we're learning, most of us prefer to do our
+math homework in pencil, not ink!
+
+If we are afraid to edit our code we won't remove complicated code that's hard
+to work with. If we are afraid to edit our code, we wont' want to try awesome
+new libraries. So, Git lets us feel free to explore!
```

Comment:
This sentence could use clarification, but I think it could start with with a more encouraging tone: "Git gives us the freedom to explore so we don't have to feel uneasy when editing code. It enables us to remove code that may be hard to work with, or even try out great new libraries!"